app/helpers/user_helper.rb

```diff
@@ -0,0 +1,31 @@
+module UserHelper
+
+  def get_users_count
+    User.count
+  end
+
+  def get_users_average_age
+    User.average(:age)
+  end
+
+  def get_youngest_user
+    User.minimum(:age)
+  end
+
+  def get_oldest_user
+    User.maximum(:age)
+  end
+
+  def get_female_count
+    User.where(sex: 'female').count
+  end
+
+  def get_male_count
+    User.where(sex: 'male').count
+  end
+
+  def get_other_count
+    User.where(sex: 'other').count
+  end
+
+end
```

Comment:
In context of MVC Helpers are part of the View. And one of the basic principles of MVC is not allowing the view to communicate with the DB. 
Considering the above let's move all the DB queries into a controller where they are supposed to live. Next you can pass the results of queries to the view using `@` variables.